lib/feature/add_character/add_character_page.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:character_viewer/feature/add_character/cubit/add_character_cubit.dart';
+import 'package:character_viewer/feature/add_character/cubit/add_character_state.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class AddCharacterPage extends StatelessWidget {
+  const AddCharacterPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Add Character'),
+      ),
+      body: BlocProvider(
+        create: (context) => CharacterCubit(),
+        child: AddCharacterForm(),
+      ),
+    );
+  }
+}
+
+class AddCharacterForm extends StatelessWidget {
+  final TextEditingController _nameController = TextEditingController();
+
+  AddCharacterForm({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<CharacterCubit, CharacterState>(
+      listener: (context, state) {},
+      builder: (context, state) {
+        return Scaffold(
+          body: _buildUI(context, state),
+        );
+      },
+    );
+  }
+
+  Widget _buildUI(BuildContext context, CharacterState state) {
```

Comment:
it's better to use Widgets, not helper methods
https://www.youtube.com/watch?v=IOyq-eTRhvo&ab_channel=Flutter